olacodes/project-manager-api
# create login and register endpoints

#### What does this PR do? 
This PR create a user login and register endpoints

#### List what gets done in this PR
- [x] create user login endpoints
- [x] cerate register user endpoint

### What left undone in this PR
- [ ] catch and format database error

#### How can this be tested
Clone this repository
Run `npm install` to install all the dependencies
Run `npm run dev` to start the app
Go to `http://127.0.0.1:4000/api/v1/auth/register` and supply the username, email and password to register
**Expected behaviour** This should return a success message and return a token

Go to `http://127.0.0.1:4000/api/v1/auth/login` and supply the username and password to log in
**Expected behaviour** This should return a success message and return a token
